polar2grid/glue.py

```diff
@@ -184,15 +198,18 @@ def write_scene(scn, writers, writer_args, datasets, to_save=None):
 
 def _handle_product_names(aliases, products):
     for prod_name in products:
-        yield aliases.get(prod_name, prod_name)
+        product = aliases.get(prod_name, prod_name)
+        if isinstance(product, DataQuery):
+            product = product['name']
```

Comment:
This is dangerous and we should be careful here. The DataQuery that this alias is using could include parameters other than name. For example, `band1_1000m` could be the 1000m resolution version of the MODIS band 1. That would be `DataQuery(name='1', resolution=1000)`. With this code that resolution information would get lost, right?

Reply:
Yes it would.  This is interesting.  I need to think about that.  That could be handled if every reader handled the name concatenation in the exact same way, but that could be a bad assumption as well.

Reply:
You mean the way you are concatenating the polarizations onto the variable name? Yeah that is definitely not how other readers work. Most readers don't have this "compound" name like MiRS does. The DataQuery objects **need** to be passed to the Scene's load. Another easy example is the `calibration` parameter. I think some of my VIIRS code has aliases where "i04_radiance" can be loaded an is an alias for `DataQuery(name='I04', calibration='radiance')`.